package.json

```diff
@@ -6,18 +6,23 @@
     "build": "fable-splitter -c splitter.config.js --define RELEASE",
     "start": "fable-splitter -c splitter.config.js -w --define DEBUG",
     "compile-for-test": "fable-splitter -c splitter.config.js --define TEST",
-    "cold-start": "fable-splitter -c splitter.config.js --define DEBUG"
+    "cold-start": "fable-splitter -c splitter.config.js --define DEBUG",
+    "reset:packager": "watchman watch-del-all && node_modules/react-native/scripts/packager.sh --reset-cache"
   },
   "dependencies": {
     "buffer": "5.0.8",
-    "core-js": "2.5.3",
-    "react": "16.2.0",
-    "react-native": "0.54.2",
-    "react-native-image-picker": "0.26.7"
+    "core-js": "2.5.7",
+    "react": "16.5.2",
+    "react-native": "0.57.0",
+    "react-native-image-picker": "0.26.10"
   },
   "devDependencies": {
+    "@babel/plugin-external-helpers": "7.0.0",
     "appium": "1.7.2",
-    "babel-core": "6.26.0",
-    "fable-splitter": "0.1.21"
+    "babel-core": "6.26.3",
```

Comment:
https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#updating-to-this-version says babel-core": "7.0.0-bridge.0" ?

Reply:
Babel dependencies are already listed in metro preset: https://github.com/facebook/metro/blob/master/packages/metro-react-native-babel-preset/package.json. Also v6.26.3 had worked for me on several projects with 0.57 so I didn't change this dependency.

Reply:
I tried to update my app similarly to your changes here. And it looks good on device, but my canopy.mobile tests fail and it looks like toolbar images are missing on the emulator

Reply:
How run to the tests?